CHANGELOG.rst

```diff
@@ -4,11 +4,11 @@ sunpy v1.1.0 (2020-01-10)
 Backwards Incompatible Changes
 ------------------------------
 
-- The `sunpy.net.vso.vso.get_online_vso_url` function has been broken into two components, the new `sunpy.net.vso.vso.get_online_vso_url` function takes no arguments (it used to take three) and now only returns an online VSO mirror or None.
-  The construction of a `zeep.Client` object is now handled by `sunpy.net.vso.vso.build_client` which has a more flexible API for customising the `zeep.Client` interface. (`#3330 <https://github.com/sunpy/sunpy/pull/3330>`__)
-- Importing `sunpy.timeseries.timeseriesbase` no longer automatically imports
+- The ``sunpy.net.vso.vso.get_online_vso_url`` function has been broken into two components, the new ``sunpy.net.vso.vso.get_online_vso_url`` function takes no arguments (it used to take three) and now only returns an online VSO mirror or None.
+  The construction of a ``zeep.Client`` object is now handled by ``sunpy.net.vso.vso.build_client`` which has a more flexible API for customising the ``zeep.Client`` interface. (`#3330 <https://github.com/sunpy/sunpy/pull/3330>`__)
```

Comment:
Is it worth adding zeep as an interlink?

Reply:
might as well.